meinberlin/react/projects/ProjectsListMapBox.jsx

```diff
@@ -55,12 +53,14 @@ const ProjectsListMapBox = ({
   searchProfilesCount,
   isAuthenticated
 }) => {
+  const location = useLocation()
+  const searchParams = new URLSearchParams(location.search)
   const [showMap, setShowMap] = useState(true)
   const [loading, setLoading] = useState(true)
-  const [projectState, setProjectState] = useState(getDefaultProjectState(searchProfile))
+  const [projectState, setProjectState] = useState(getDefaultProjectState(searchParams))
   const [items, setItems] = useState([])
   const fetchCache = useRef({})
-  const [appliedFilters, setAppliedFilters] = useState(getDefaultState(searchProfile))
+  const [appliedFilters, setAppliedFilters] = useState(getDefaultState(searchParams, { districts, organisations, participationChoices, topicChoices }))
```

Comment:
The state is already there in the search params now, why not `const [state] = useSearchParams`?